.ci/deploy-mac.sh

```diff
@@ -15,7 +15,7 @@ echo "AVVER=$AVVER" >> ../.ci/ci-vars.env
 cd bin
 mkdir "rpcs3.app/Contents/lib/"
 
-cp "/usr/local/Homebrew/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
+cp "/usr/local/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
```

Comment:
```suggestion
ARCH=$(uname -p)
if [ $ARCH = "arm" ]; then
	cp "/opt/homebrew/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
else
	cp "/usr/local/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
fi
```

Reply:
I added double quotes around the `$CMAKE_EXTRA_OPS` and `$ARCH` as advised by the compiler. It should still work and will fix the linux builds

Reply:
Thank you

Reply:
Hmm it wont let me commit it

Reply:
I’m an idiot. We should be using `$(brew --prefix)` here

Reply:
oh yeah bc it will be different based on using rosetta2 or not

Reply:
its doing that dumb thing where it still wont let me commit ur suggestion ;-;

Reply:
"outdated suggestions cannot be applied"

Reply:
Maybe just close this comment and apply it manually?